docs/source/user-guide/tabular/supported_models.rst

```diff
@@ -1,107 +1,111 @@
+...     @property
+...     def feature_importances_(self):
 ...         ...
 
-Instead of a model you can also pass the model predicted probabilities as keyword arguments to the run function as follows:
-
->>> train_proba = model.predict_proba(train_dataset.features_columns)
-... test_proba = model.predict_proba(test_dataset.features_columns)
-... 
-... suite.run(train_dataset=train_dataset, test_dataset=test_dataset,
-...           features_importance=feature_importance,
-...           y_proba_train=train_proba, y_proba_test=test_proba)
+Running Deepchecks with a Supported Model
+=========================================
 
-You may also provide the predictions as was done in the Regression example (by default the predictions will be assumed to be the argmax of the probability matrix)
-For more info check out :doc:`Using Pre-computed Predictions </user-guide/tabular/tutorials/plot_static_predictions>`
+>>> from deepchecks.tabular.datasets.classification.iris import load_data, load_fitted_model
+... from deepchecks.tabular.suites import model_evaluation
+...
+... ds_train, ds_test = load_data(data_format='Dataset')
+... rf_clf = load_fitted_model() # trained sklearn RandomForestClassifier
+... model_evaluation().run(train_dataset=ds_train, test_dataset=ds_test, model=rf_clf)
 
+Using Pre-computed Predictions
+******************************
 
-.. _supported_models__optional_model_interface:
+There are several cases in which it's useful to provide pre-computed predictions instead of a model object.
 
-Optional Model Interface 
-===========================
+For example, in cases where the model's prediction process takes a long time or is computationally expensive.
+Another example is when the model is stored in different location or does not have a compatible API for providing
+the predictions.
 
-Feature Importance
--------------------
-
-Deepchecks can calculate feature importance using |permutation importance|, and it also supports the builtin feature importance property: ``feature_importances_`` or ``coef_`` for a linear model.
-The default behavior is to use the builtin feature importance property if it exists, and if it doesn't, we calculate the feature importance using permutation importance.
-
-
->>> class simple_importance_model:
-...     def predict(X: pd.DataFrame) -> pd.Series:
-...         ...
-...     @property
-...     def feature_importances_(self):
-...         ...
-
-Check-Specific Model Interfaces
---------------------------------
+.. Note::
+    If the train dataset shares indices with the test dataset we will add train/test prefixes to the indexes.
+
+
+Running Deepchecks with Pre-computed Predictions
+================================================
+
+We will run the Deepchecks model evaluation suite using static predictions from a random forest classification model.
+Since it is a classification model we will supply the ``proba``  probabilities predictions as defined above.
```

Comment:
same here about ``proba``

Reply:
"as defined above" - they probably didn't read, since it's under a "Using Model Interface" section.
Possibly - direct link + separate the "required api" and the "supported model / wrapper" vs. "model predictions" options?